inspirehep/modules/records/utils.py

```diff
@@ -414,6 +420,25 @@ def populate_authors_full_name_unicode_normalized(json):
         })
 
 
+def populate_facet_author_name(json):
+    """Populate the ``facet_author_name`` field of Literature records."""
+    linked_authors = list(get_linked_records_in_field(json, 'authors.record'))
```

Comment:
should be already a list, if not we use ``from inspire_utils.helpers import force_list``

Reply:
I'd name these ``authors_with_record`` and ``authors_without_record`` to be more clear. WDYT?